package.json

```diff
@@ -57,14 +57,14 @@
     },
     {
       "author": "magik6k",
-      "hash": "QmUCfrikzKVGAfpE31RPwPd32fu1DYxSG7HTGCadba5Wza",
+      "hash": "QmaiEBFgkgB1wjrPRxru5PyXPEkx58WuMjXNaR1Q9QNRjn",
       "name": "go-ds-badger",
```

Comment:
depending on badger directly this far down is a bit weird. Especially since its not something we are tied to at this point (and may move away from)

Reply:
It's just used in tests to create test instances -- not in production code. Any other way you can think of?

Reply:
In Java, this dependency would be in `test` scope in maven or gradle. No such thing in Go / gx AFAIK.

Reply:
not a gx thing. `go get` by default won't download test dependency packages. The `-t` flag is required to download test deps.

Reply:
How difficult would it be to add *basic* ttl support to the in-memory datastore. It doesn't have to be fancy, it could literally be a:

1. Check on read.
2. Sweep every N minutes.

Reply:
You mean the _datastore_ not the _peerstore_, right? It would be pretty straightforward, I'll work on it!

Eventually (when KeyBook and PeerMetadata are migrated to use the datastore), it may make sense to remove the in-memory implementation of the _peerstore_ and it delegate to a map-based datastore peerstore.

_That's almost a tongue twister!_